### PR TITLE
Update MicrosoftTeams.schema.json

### PR DESCRIPTION
**Pull request by MattSFT**

### CODE REVIEW

File: MicrosoftTeams.schema.json [hunk 1]
```diff
@@ -369,79 +369,16 @@
...
-                                                "maxLength": 128
-                                            }
-                                        },
-                                        "required": [
-                                            "name",
-                                            "title"
-                                        ]
-                                    }
+                                {
+                                    "$ref": "#/definitions/actionParamCommand"
```
Comment (hamenon-ms) [inline MicrosoftTeams.schema.json]: Nit pick, but shouldn't just be commandParam or something? Cause technically params are associated with query as well

File: MicrosoftTeams.schema.json [hunk 2]
```diff
@@ -518,5 +455,190 @@
...
+                        },
+                        "required": [
+                            "name",
+                            "title"
+                        ]
+                    }
+                }
+            },
+            "required": [
+                "type",
```
Comment (hamenon-ms) [inline MicrosoftTeams.schema.json]: Shouldn't id be required too?

File: MicrosoftTeams.schema.json [hunk 3]
```diff
@@ -518,5 +455,190 @@
...
+                        },
+                        "required": [
+                            "name",
+                            "title"
+                        ]
+                    }
+                }
+            },
+            "required": [
+                "parameters"
```
Comment (hamenon-ms) [inline MicrosoftTeams.schema.json]: Command id and title should be required too

File: MicrosoftTeams.schema.json [hunk 4]
```diff
@@ -518,5 +455,190 @@
...
+                        "required": [
+                            "name",
+                            "title"
+                        ]
+                    }
+                }
+            },
+            "required": [
+                "type",
+                "parameters"
```
Comment (hamenon-ms) [inline MicrosoftTeams.schema.json]: Same as before, id and title should be required too

**Comment by hamenon-ms** [comment]
@robin-liao can you also have a look at this PR?

File: MicrosoftTeams.schema.json (hunk 2 above)
Comment (MattSFT) [reply MicrosoftTeams.schema.json]: yes

File: MicrosoftTeams.schema.json (hunk 4 above)
Comment (MattSFT) [reply MicrosoftTeams.schema.json]: yes

**Comment by robin-liao** [comment]
If we do this change, then when should we also do it for devPreview? Otherwise, when the devPreview merge into master, it'll probably overwrite to old one? I mean, why not we opt in this change in devPreview instead of master? So next release of snapping we can just override the master content instead of resolving conflicts? 

@MattSFT @hamenon-ms ?

**Pull request merged**